### PR TITLE
Create Weekly Announcement for 2021-06-04

### PR DESCRIPTION
**Pull request by cokiengchiara**
### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes

Notes
* There are no app-related changes

### CODE REVIEW

File: src/content/changes/highlights/2021-06-04.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+title: Highlights for the week ending Jun 06, 2021
+changes_categories:
+- Highlights
+owner:
+- https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## User interfaces
+
+We fixed issues related to preparing for providing Management API access. These include correcting the chart name when installing an app and adding a missing import in the ingress installation step. We also upgraded the `ws` dependency to address a security vulnerability.
```
Comment (cokiengchiara) [inline src/content/changes/highlights/2021-06-04.md:12]: Not sure if these are 'minor' but I think they are worth including. After we're all about iteration.

File: src/content/changes/highlights/2021-06-04.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+## Documentation
+
+We wrote an article on [Organizations](https://docs.giantswarm.io/general/organizations/), as defined in the Management API. These replace the organizations as used with the REST API. Please read the details of this transition carefully to understand the ramifications for you and your end users.
+
+These weekly announcements now also appear in the Changes and Releases section, as their own category named [Highlights](https://docs.giantswarm.io/changes/highlights/).
```
Comment (cokiengchiara) [inline src/content/changes/highlights/2021-06-04.md:18]: This appears 2nd time now, but I think it's worth repeating, maybe up to 4x (?)

Comment (marians) [reply src/content/changes/highlights/2021-06-04.md:18]: I think after this one, we are fine. This is not critical info, so if some people miss this, that's fine.

Comment (cokiengchiara) [reply src/content/changes/highlights/2021-06-04.md:18]: ok!

**Review by marians — APPROVED** [review]

**Pull request merged**